pints/_log_likelihoods.py

```diff
@@ -117,6 +117,120 @@ def __call__(self, x):
                       - np.sum(autocorr_error**2, axis=0) / (2 * sigma**2))
 
 
+class GaussianIntegratedUniformLogLikelihood(pints.ProblemLogLikelihood):
```

Comment:
When would I use this method, what advantages does it have over the normal Gaussian one?

Reply:
@MichaelClerx See the additional information I've added in the docstrings (also in the example notebook) to justify why one might want to use this distribution.